api/chaos/serverexperimentation/v1/serverexperimentation.proto

```diff
@@ -56,3 +56,83 @@ enum FaultInjectionCluster {
   // Faults are injected to ingesss traffic by an upstream service(s).
   FAULTINJECTIONCLUSTER_UPSTREAM = 2;
 }
+
+message HTTPFaultConfig {
+  oneof fault {
+    option (validate.required) = true;
+
+    Abort abort = 1;
+    Latency latency = 2;
+  }
+}
+
+message Abort {
+  FaultTargeting fault_targeting = 1 [ (validate.rules).message.required = true ];
+  FaultPercentage fault_parcentage = 2 [ (validate.rules).message.required = true ];
```

Comment:
any opinion on removing `fault_` from the name of the field here? @snowp

Reply:
sgtm. `percentage`, not `parcentage`